feature-group-definitions/fetch-metadata-request-headers.yml

```diff
@@ -0,0 +1,8 @@
+name: Fetch metadata request headers
+description: The headers `Sec-Fetch-Dest`, `Sec-Fetch-Mode, `Sec-Fetch-Site`, and `Sec-Fetch-User` provide extra information about the way a request was made, to help servers reject certain kinds of malicious requests.
+spec: https://w3c.github.io/webappsec-fetch-metadata/
```

Comment:
Let's make the feature identifier `fetch-metadata` to match https://www.w3.org/TR/fetch-metadata/. (Even when TR isn't published, it's always been the repo name without `webappsec-` in cases I remember.)